bin/tapestry-build

```diff
@@ -6,13 +6,51 @@ const exec = require('child_process').exec
 const logger = require('../dist/logger')
 const cwd = process.cwd()
 const path = require('path')
```

Comment:
Small thing – `path` and `child_process` modules aren't being used anymore – can probably remove from this file